### ISSUE TITLE
Feature request: Test grammar rules only

### ISSUE THREAD

**Issue by agershun** [nolanlawson/jison-debugger#3]
@nolanlawson  May I ask you to add a checkbox to check only grammar rules without JavaScript internal code. For example:

Source grammar:

```
    DeclaresList
    : DeclareItem
        { $$ = [$1]; }
    | DeclaresList COMMA DeclareItem
        { $1.push($3); $$ = $1; }
    ;
```

Debugging grammar:

```
     DeclaresList
    : DeclareItem
    | DeclaresList COMMA DeclareItem
    ;
```

**Comment by nolanlawson** [comment]
Hm, I think unfortunately this might not be possible, given how the Jison debugger is designed.

Notice that it shows you an intermediate result for each node (when you hover over). This is useful for things like the calculator, where it's slowly building up a final result.

I take it your problem is that the AlaSQL grammar isn't working in the Jison Debugger? I actually tried to copy-paste your code in (it was very instructive for me! :smiley:), but I noticed it didn't work because of dependencies on external JavaScript code.  Maybe a better solution would be to allow users to include this external code?

**Comment by agershun** [comment]
Exactly! I just copied AlaSQL and WebSQLShim Jison files,and got the same result. After I wrote the issue, I think, that it requires separate parser to parse Jison files to eliminate JavaScript code and this is separate complex work... Probably it is not worth it..
